src/test/script_segwit_tests.cpp

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2012-2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <script/script.h>
+#include <test/util/setup_common.h>
+
+#include <boost/test/unit_test.hpp>
+
+BOOST_FIXTURE_TEST_SUITE(script_segwit_tests, BasicTestingSetup)
```

Comment:
nit: I don't think you need any setup. `BOOST_AUTO_TEST_SUITE(script_segwit_tests)` should work fine for pure utility tests.